doc/theoretical_background.rst

```diff
@@ -7,158 +7,112 @@ Theoretical Background
 Definitions and Units
 =====================
 
-eTraGo executes the Open Source software PyPSA to perform power flow 
-simulations and uses its definitions and
-`units <https://pypsa.org/doc/conventions.html#unit-conventions>`_.
-
+eTraGo is based on the open source tool `PyPSA <https://pypsa.readthedocs.io/en/latest/>`_ and uses its definitions and units.
 
 
 Assumptions on Data
 ===================
 
-eTraGo fetches its necessary input data from the OpenEnergy Platform including
-load, generation, grid and scenario-related data. More details can be found in
-the `Data-Processing <https://data-processing.readthedocs.io/en/latest>`_.
+eTraGo fetches the input data from the `OpenEnergy Platform <https://openenergy-platform.org/>`_. The data includes electricity and gas grid topology as well as data on energy supply and load for the considered sectors (electricity, gas, heat and e-mobility) and data on additional electrical flexibilities such as Dynamic Line Rating or Demand Side Management. More details on the data model can be found in the documentaton of `eGon-data <https://egon-data.readthedocs.io/en/latest/>`_.
```

Comment:
The sectors heat, gas and mobility also provide flexibility options, which I can not really find in these sentences.